apps/s_client.c

```diff
@@ -15,6 +15,7 @@
 #include <string.h>
 #include <errno.h>
 #include <openssl/e_os2.h>
+#include <openssl/provider.h>
```

Comment:
This should not be needed?

Reply:
Indeed, I'll remove that. It was a leftover from my earlier joke with the provider test.